src/key/eddsa.ts

```diff
@@ -0,0 +1,74 @@
+import * as _ from "lodash";
```

Comment:
Could you rename the file name "ed25519.ts"?

Reply:
I modified it.